### DISCUSSION TITLE
Pass through options to prime ng

### DISCUSSION THREAD

**Discussion by DAVIAMERICO242** [primefaces/community#2829] [PrimeNG]
Pass through options to prime ng

**Comment by qwadrox** [comment]
@DAVIAMERICO242 
Not sure if this is a question or a feature request. Next time please elaborate more on your inquiries. We can only give a quality answer if you create a quality post.

"Pass through props" were planned in Q1 of this year but the roadmap has changed and the team probably decided to focus on it later.
Most likely it's going to come but only next year.
We can consider PrimeVue as their state of the art library, meaning everything PrimeVue has, will be implemented in the other libs as well. Since PrimeVue has Pass through props we can expect here in PrimeNg also.

https://github.com/primefaces/primeng/blob/d6c7ffcfc5cb9a8ed355399ddae276c279836ccf/src/app/showcase/pages/roadmap/roadmap.component.html#L32
